repos/dir.targets

```diff
@@ -219,5 +217,14 @@
   <Target Name="GetProjectDirectory" Outputs="$(ProjectDirectory)" />
   <Target Name="GetOrchestratedManifestBuildName" Outputs="$(OrchestratedManifestBuildName)" />
 
+  <Target Name="DisablePatchedRepoSubmoduleAutoUpdate"
```

Comment:
I think we're going to have repos that we're going to want to update, even though they have patches.  Is there a different way to trigger the disable of auto update?  Or should we have an override for those repos?

Reply:
I was thinking patches are brittle enough that they would break upgrade PR CI often, so we'd never want to upgrade them. The `<OrchestratedManifestBuildName>N/A</OrchestratedManifestBuildName>` can certainly be inlined into the specific repo.proj files though.

Reply:
We talked offline--I'm going to let these update. The idea is that we'll be able to see when the patches break and fix them at that point, so we won't get far behind.

(I'll make this change once the CoreFX PR goes in and I resolve the conflicts.)